bound/kt/pom.xml

```diff
@@ -188,6 +188,11 @@
                 <artifactId>kotlinx-coroutines-core</artifactId>
                 <version>${version.org.jetbrains.kotlinx.kotlinx.coroutines.core}</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.commons</groupId>
+                <artifactId>commons-lang3</artifactId>
+                <version>3.16.0</version>
```

Comment:
Would you mind dropping the version in a property at the top like we do for the other deps? Helps for future maintainers to do version upgrades especially in cases where the version is reused

Reply:
as you discovered @ALRubinger apache commons was doing the same thing we were previously doing https://github.com/apache/commons-lang/blob/master/src/main/java/org/apache/commons/lang3/SystemProperties.java#L1307C5-L1307C52

rolled back the git history to undo this change; we prefer not to add unnecessary dependencies